slack_cli_hooks/hooks/check_update.py

```diff
@@ -30,8 +30,8 @@ def __init__(
     ):
         self.name = name
         if current and latest:
-            self.current = current.base_version
-            self.latest = latest.base_version
+            self.current = str(current)
+            self.latest = str(latest)
             self.update = current < latest
             self.breaking = (current.major - latest.major) != 0
```

Comment:
Could a test also be added for this? We might want to also avoid major downgrades(??) with

```
latest.major - current.major > 0
```

Reply:
Yeah, I was also wondering about the breaking property's behavior. Adding more various patterns to the test is safer. Other than that, this PR looks good to me

Reply:
Good catch 💯 not sure how this got in, fixed and added more tests around it